state/syncer/userAccountsSyncer.go

```diff
@@ -252,11 +256,10 @@ func (u *userAccountsSyncer) syncAccountDataTries(
 			defer u.throttler.EndProcessing()
 
 			log.Trace("sync data trie", "roothash", trieRootHash)
-			newErr := u.syncDataTrie(trieRootHash, address, ctx)
-			if newErr != nil {
-				errMutex.Lock()
-				errFound = newErr
-				errMutex.Unlock()
+			err := u.syncDataTrie(trieRootHash, address, ctx)
+			if err != nil {
+				log.Error("sync data trie: error found", "roothash", trieRootHash, "error", err.Error())
```

Comment:
instead of this log print we can return err here directly.
Reason: if we encounter error on any data trie, the process will fail anyway because of L146. So why wait trying to sync all data tries if one failed?

Reply:
i have to send it to ErrChan since it's on a separate goroutine, will think for a solution here

Reply:
removed log entry